notebooks/feature_engineering/raw/tut1.ipynb

```diff
@@ -6,18 +6,18 @@
    "source": [
     "# Feature Engineering\n",
     "\n",
-    "Welcome to this mini-course on feature engineering. The goal here is to help you learn a practical approach to feature engineering in as little time as possible. You'll be able to apply what you learn to Kaggle competitions as well as building machine learning applications. Through hands-on exercises you'll learn how to:\n",
+    "In this micro-course, you will learn a practical approach to feature engineering. You'll be able to apply what you learn to Kaggle competitions as well as building machine learning applications. Through hands-on exercises you will:\n",
     "\n",
     "* Develop a baseline model for comparing performance on models with more features\n",
     "* Encode categorical features so the model can make better use of the information\n",
-    "* Generate new features from the data to provide more information for the model\n",
-    "* Select features to reduce overfitting and increase the speed of making predictions\n",
+    "* Generate new features to provide more information for the model\n",
+    "* Select features to reduce overfitting and increase prediction speed\n",
     "\n",
-    "For the exercises, you'll be applying feature engineering techniques to data from the TalkingData AdTracking competition.\n",
+    "In the exercise, you will apply feature engineering techniques to data from the TalkingData AdTracking competition.\n",
     "\n",
     "<center><a href=\"https://www.kaggle.com/c/talkingdata-adtracking-fraud-detection\"><img src=\"https://i.imgur.com/srKxEkD.png\" width=600px></a></center>\n",
     "\n",
-    "The goal of the competition is to predict if a user will download an app after clicking through an ad. For this course I've taken a small sample of the data, dropping 99% of negative records (where the app wasn't downloaded) so that the target is more balanced."
+    "The goal of the competition is to predict if a user will download an app after clicking through an ad. For this course you will use a small sample of the data, dropping 99% of negative records (where the app wasn't downloaded) to make the target more balanced."
```

Comment:
As part of our style guide, we try to phrase things in the second person where possible (and avoid the first person singular).